runtime/bcutil/ClassFileOracle.hpp

```diff
@@ -68,6 +68,9 @@ class ClassFileOracle
 		J9CfrAttributeRuntimeVisibleAnnotations *annotationsAttribute;
 		J9CfrAttributeRuntimeVisibleTypeAnnotations *typeAnnotationsAttribute;
 		bool isFieldContended;
+#if defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES)
```

Comment:
The new macros (`IMPLICIT_CREATION_FLAGS_DEFAULT` and `IMPLICIT_CREATION_FLAGS_NON_ATOMIC`) added in line 49 and 50 can be put inside `if defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES) `as well.

Reply:
fixed